scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.42.5] - 2020-07-8
+### Added
+- Port mapping support for k8s, from @dposada
+- Total number of pods and nodes quota for k8s, from @scrosby
+### Changed
+- Added extra metrics and logging around match cycle, from @scrosby
```

Comment:
This was already in 1.42.4, right?

Reply:
It is. Copy&paste error. Fixed and repushed.